### PR TITLE
doc: Add a Quick Start section to the README

### PR DESCRIPTION
**Pull request by lgarber-akamai**
## 📝 Description

This change adds a quick start section to the project's README file. This section should reduce friction for new users by providing a usage example without the need to read through the full Getting Started guide.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,11 +30,60 @@ To build and install this package:
 - ``./setup.py install``
 
 Usage
------
+=====
+
+Quick Start
+-----------
+
+In order to authenticate the with the Linode API, you will first need to create a
```
Comment (ezilber-akamai) [inline README.rst:38]: Looks like you have a typo here.

**Review by ezilber-akamai — CHANGES_REQUESTED** [review]

File: README.rst [hunk 2]
```diff
@@ -30,11 +30,60 @@ To build and install this package:
...
+`Linode Personal Access Token`_ with your desired account permissions.
+
+The following code sample can help you quickly get started using this package.
+
+.. code-block:: python
+
+    from linode_api4 import LinodeClient, Instance
+
+    # Create a Linode API client
+    client = LinodeClient("MYPERSONALACCESSTOKEN")
```
Comment (yec-akamai) [inline README.rst:48]: nit: the value maybe more readable in a snake or camel case?

File: README.rst [hunk 3]
```diff
@@ -30,11 +30,60 @@ To build and install this package:
...
+
+    # Create a new Linode
+    new_linode, root_pass = client.linode.instance_create(
+        "g6-nanode-1",
```
Comment (yec-akamai) [inline README.rst:52]: nit: may worth also specifying the attributes' names here so a new user can better understand, i.e. 
```
type="g6-nanode-1",
region="us-southeast",
```

**Review by yec-akamai — APPROVED** [review]
Thank you for adding this session! I think it clarifies a lot for a new user to get started ;)

**Review by ezilber-akamai — APPROVED** [review]

**Pull request merged**